tests/vcs/git/GitCommitBearTest.py

```diff
@@ -288,6 +288,24 @@ def test_body_checks(self):
                              body_regex=r'TICKER\s*CLOSE\s+[1-9][0-9]*'), [])
         self.assert_no_msgs()
 
+        # Testing invalid emails
+        self.git_commit('Shortlog\n\n'
+                        'invalid#!format!*!@email.com\n'
+                        'another.*invalid@format\n'
+                        'validformat@email.check\n'
+                        'validdomain@gmail.com\n'
+                        'Fix 2017')
+        message = 'Body contains these invalid emails:\n'
+        message += ' invalid#!format!*!@email.com\n'
+        message += ' another.*invalid@format\n'
+        # Test with email_dns_check set to False
+        self.assertEqual(self.run_uut(), [message])
+        # Test with email_dns_check set to True
+        message += " validformat@email.check\n"
```

Comment:
You do not use the preferred quotation marks.

*QuotesBear, severity NORMAL, section `python`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tests/vcs/git/GitCommitBearTest.py
+++ b/tests/vcs/git/GitCommitBearTest.py
@@ -301,7 +301,7 @@
         # Test with email_dns_check set to False
         self.assertEqual(self.run_uut(), [message])
         # Test with email_dns_check set to True
-        message += " validformat@email.check\n"
+        message += ' validformat@email.check\n'
         self.assertEqual(self.run_uut(email_dns_check=True), [message])
         # Test with valid_email_check set to False
         self.assertEqual(self.run_uut(valid_email_check=False), [])
```